src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -1676,8 +1708,18 @@ protected function get_attendees_by_order_id( $order_id, $ticket_id = null ) {
 	 * @return array
 	 */
 	public function get_order_data( $order_id ) {
-		$name       = get_post_meta( $order_id, $this->full_name, true );
-		$email      = get_post_meta( $order_id, $this->email, true );
+		$name  = get_post_meta( $order_id, $this->full_name, true );
+		$email = get_post_meta( $order_id, $this->email, true );
+
+		$order = Tribe__Tickets__Commerce__PayPal__Order::from_attendee_id( $order_id, [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 3864c93ebd779a8505c785761ec9fd21</sub>